### DISCUSSION TITLE
Are you using ChatGPT/LLMs with Storybook?

### DISCUSSION THREAD

**Discussion by joevaugh4n** [storybookjs/storybook#24917] [General]
Hey there! We've shared a couple of examples on Storybook's X account in the last few days of how Storybook users are bringing ChatGPT into Storybook:

- [@kaelig's StorybookGPT](https://x.com/storybookjs/status/1726604084801872072?s=20) – automatically create stories for React+TS components
- [Eduard Constantin's Storybook Genie](https://github.com/eduardconstantin/storybook-genie) – generate stories for React components via CLI!

Are you using ChatGP/AI to power up your Storybook? If so, I'd love to know what you're doing, why, and how!

Thanks!

**Comment by getflourish** [comment]
Unfortunately most GPTs struggle to access Storybook due to its client-side rendering. 

Any solutions? Addons that generate llms.txt?

**Comment by lerenos** [comment]
I'm right now trying to make a llms.txt file from a storybook, so I can instruct v0.dev to repect the component library I want to use (https://storybook.heroui.com/, for example).
It would be great to see an automatic llms.txt generator from storybook!

**Reply by svdoever** [reply]
Any success in generating llms.txt file and connected Markdown files?

**Reply by dmytrokirpa** [reply]
We have some early draft PR for FluentUI and Storybook 7 https://github.com/microsoft/fluentui/pull/34838

**Reply by dervondenbergen** [reply]
@dmytrokirpa just wondering as your mentioned PR was merged: do you plan on publishing the package on npm (anytime soon) or should I try to figure out how to install the tool as a package out of this repo?

**Reply by dmytrokirpa** [reply]
> @dmytrokirpa just wondering as your mentioned PR was merged: do you plan on publishing the package on npm (anytime soon) or should I try to figure out how to install the tool as a package out of this repo?

We'd definitely plan publishing it to npm, regarding the timeline, probably sometime next week. I'll update the discussion once it's published

**Reply by getflourish** [reply]
👀

**Reply by dmytrokirpa** [reply]
UPD: the package has been published on [NPM](https://www.npmjs.com/package/@fluentui/storybook-llms-extractor)

**Comment by micahgodbolt** [comment]
[celebrate]     Micah Godbolt reacted to your message:
________________________________

**Comment by xyy94813** [comment]
What should i do, if i want my site that is build by storybook is friendly to llms?

`vitepress-plugin-llms` is awesome

**Comment by shilman** [comment]
For anybody who stumbles across this we are building an MCP server, which we announced on our blog: https://storybook.js.org/blog/storybook-mcp-sneak-peek/

We have stuff working already (and it's open source at https://github.com/storybookjs/mcp), usable today but still poorly documented. We'll be documenting it more and releasing it officially once we've validated it with a bunch of teams on real-world workflows.

For anybody who wants a static asset instead of an MCP server, the same metadata we're gathering can be easily used to generate an `llms.txt` file. More soon!

**Reply by xiongemi** [reply]
@shilman i am just wondering, how to easily generate llms.txt file? is the solution out?

**Reply by shilman** [reply]
@xiongemi should be pretty easy to generate from the manifest. @JReinhold do we have any docs on the manifest structure?

**Reply by xiongemi** [reply]
@shilman thanks. will try to generate from manifest.

**Reply by JReinhold** [reply]
We don't @shilman, as we still consider it an internal construct and not a public API.

**Reply by KamilEmeleev** [reply]
@shilman Nice idea, thanks! I'm using the same approach: taking the metadata from the MCP server and turning it into plain text for `llms.txt`. Here's the first implementation of [script](https://github.com/koobiq/react-components/blob/main/scripts/generate-llms-txt.mjs).
